src/coreclr/src/vm/ClrEtwAll.man

```diff
@@ -710,6 +710,43 @@
                         </UserData>
                     </template>
 
+                    <template tid="RuntimeProperties">
+                        <data name="ClrInstanceID" inType="win:UInt16" />
+                        <data name="TrustedPlatformAssemblies" inType="win:UnicodeString"/>
```

Comment:
There is nothing `Trusted` not `Platform` about these. Should we use this as an opportunity to come up with more sensible name that does not need 5 minute explanation?

Reply:
I agree, we should choose a better name, since it is just a list of assemblies from which we resolve dependencies by default. I spoke to @richlander  about this, he said he'll suggest a better name.

Reply:
I have heard about the TPA list for many years, but I don't know how it fits into frameworks and other .NET Core-isms. Can you describe what it means to you?

Reply:
- Is is a list of assemblies that the application is composed of.
- It is a list of assemblies that are (logically) pre-loaded into default assembly load context.

Some naming ideas: `Assemblies`, `DefaultAssemblies`

Reply:
Good suggestions. Those help. They are also a bit misleading. For example, DefaultAssemblies doesn't align with the [default AssemblyLoaderContext](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.loader.assemblyloadcontext.default?view=netcore-3.1). That would be confusing to me.

When a dependency needs to be resolved from a non-default ALC by the default ALC, I assume assemblies other than the TPA ones would be candidates. Is that right?

I take it that the TPA is basically the set of "pre-bound assemblies."  How about `ApplicationAssemblies`.

Reply:
> When a dependency needs to be resolved from a non-default ALC by the default ALC, I assume assemblies other than the TPA ones would be candidates

Maybe I'm missing what you mean by this, but if the default ALC needs to resolve a dependency that is not on the TPA list, the runtime would not know where to look for that assembly? The user would need to have added some sort of event handler (AssemblyLoadContext.Resolving or AppDomain.AssemblyResolve) to do the resolution themselves.

Reply:
>> When a dependency needs to be resolved from a non-default ALC by the default ALC, I assume assemblies other than the TPA ones would be candidates. Is that right?

When an assembly is loaded into non-default ALC, the ALC's load method tries to resolve the assembly using custom logic in its Load() overrides. If the assembly cannot be resolved that way, the resolution defers to the default-ALC, which tries to resolve the assembly using the TPA list. So, the TPAs are closely associated with the default-ALC.

>> How about `ApplicationAssemblies`.

`ApplicationAssemblies` sounds like a good name, because typically the application's assemblies are loaded into the default-ALC, while the plugins are loaded into custom-ALC.

Reply:
I think @elinor-fung used the term `DefaultAssemblies` in the other PR, which I actually like better. There are two sets of assemblies which are typically present in the TPA:
* The frameworks
* Application assemblies

Both are equally important when dealing with plugins and such. Calling it just `ApplicationAssemblies` might mean to someone that it's just the app stuff, not the shared frameworks.
But it's still much better than TPA :-)